src/test/java/org/jenkinsci/plugins/dependencycheck/model/ReportParserTest.java

```diff
@@ -0,0 +1,17 @@
+package org.jenkinsci.plugins.DependencyCheck.model;
+
+import static org.junit.Assert.fail;
+
+import java.lang.reflect.InvocationTargetException;
+
+import org.junit.Test;
+
+public class ReportParserTest {
+    @Test(expected = InvocationTargetException.class)
+    public void testRejectsExternalEntities() throws Exception
+    {
+        new ReportParser(1).parse(getClass().getResourceAsStream("dependency-check-report-external-entities.xml"));
```

Comment:
It looks like this test is failing because "Import Stream is null"

![image](https://user-images.githubusercontent.com/3792426/142274678-f7748786-4e5f-4810-ade3-3f2d34f5725b.png)

Reply:
This was the case mismatch between the code and the file structure. Fixed.